### ISSUE TITLE
Accessibility: Swipe elements not obvious and hard to use with talkback

### ISSUE THREAD

**Issue by davidbtc2009** [cake-tech/cake_wallet#564]
It is difficult to know what will respond to swipes. As an example, i saw no way of deleting a node until just now when i tried a swipe. I restored my wallet from seed (to make sure i could) and as expected my sub accounts were untitled. It took me reading an old bug report here on github to consider that it might be a swipe element. Also, these are not accessible with talkback as the edit, delete, etc are not in the talkback actions menu.
